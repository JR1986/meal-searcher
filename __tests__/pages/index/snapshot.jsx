import React from 'react';
import { render } from '@testing-library/react';
import Search from '../../../pages/index';

it('renders search page', () => {
  const { container } = render(<Search />);
  expect(container).toMatchSnapshot();
});
