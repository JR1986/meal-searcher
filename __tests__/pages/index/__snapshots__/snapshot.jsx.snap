// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders search page 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
  >
    <h1
      class="MuiTypography-root MuiTypography-h3 MuiTypography-alignCenter css-9mg3ao-MuiTypography-root"
    >
      Search Meal
    </h1>
    <div
      class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
    >
      <div
        class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl css-md26zr-MuiInputBase-root-MuiOutlinedInput-root"
      >
        <input
          aria-invalid="false"
          class="MuiOutlinedInput-input MuiInputBase-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
          id="mui-1"
          placeholder="Search meal"
          type="text"
          value=""
        />
        <fieldset
          aria-hidden="true"
          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <legend
            class="css-nnbavb"
          >
            <span
              class="notranslate"
            >
              â€‹
            </span>
          </legend>
        </fieldset>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4 css-12urgrj-MuiGrid-root"
    >
      <p>
        No result
      </p>
    </div>
  </div>
</div>
`;
